nextflow.config

```diff
@@ -149,6 +149,28 @@ params {
     damageprofiler_length        = 100
     damageprofiler_threshold     = 15
     damageprofiler_yaxis         = 0.3
+
+    // Damage Manipulation options
+    run_mapdamage_rescaling                                          = false
+    damage_manipulation_rescale_seqlength                            = 12
+    damage_manipulation_rescale_length_5p                            = 0
+    damage_manipulation_rescale_length_3p                            = 0
+    run_pmd_filtering                                                = false
+    damage_manipulation_pmdtools_threshold                           = 3
+    damage_manipulation_pmdtools_reference_mask                      = null
+    run_trim_bam                                                     = false
+    damage_manipulation_bamutils_clip_double_stranded_none_udg_left  = 0
+    damage_manipulation_bamutils_clip_double_stranded_none_udg_right = 0
+    damage_manipulation_bamutils_clip_double_stranded_half_udg_left  = 0
+    damage_manipulation_bamutils_clip_double_stranded_half_udg_right = 0
+    damage_manipulation_bamutils_clip_single_stranded_none_udg_left  = 0
+    damage_manipulation_bamutils_clip_single_stranded_none_udg_right = 0
+    damage_manipulation_bamutils_clip_single_stranded_half_udg_left  = 0
+    damage_manipulation_bamutils_clip_single_stranded_half_udg_right = 0
+    damage_manipulation_bamutils_softclip                            = false
```

Comment:
Replace `clip` with `trim` for consistency with `--run_trim_bam`, I wonder if you can make the parameters shorter, e.g.

`--damagemanipulation_bamutils_double_noneudg_right`

Thinking about it bamUtils does nothing other than trimming, so trim is not even necessary, note the snake case underscores only apply to separate 'sections' of the parameter name, not replacing every spce.

Reply:
ok for the first. But I'm not onboard with the second. Currently the trimming parameters are called `bamutils_clip_double_stranded_half_udg_left`. Adding `_` for section only makes the parameter far less legible imo. Also not sure why we'd omit `stranded` but not `udg` in your example (unless typo?).